db/seeds.rb

```diff
@@ -5,3 +5,468 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Daley', city: cities.first)
+
+require 'faker'
+require 'ruby-progressbar'
+################################################################################
+## Parameters
+################################################################################
+# Number of records
+NLoc_group = 3
+NLocation_per_group = 3
+NUser = 50
+NAnnouncement = 6
+NSticky = 10
+NLink = 10
+NNotice_department_wide = 1
+NShift_per_day_per_location = 3
+NTasks_per_location = 5
+NData_Object = 5
+NCategories = 3
+NPayform_item_per_user = 3
+NReport_item_per_report = 5
+
+# Other params
+Ndays_of_schedule_into_past = 7
+Ndays_of_schedule_into_future = 14
+Shifts_duration_min_hours = 1
+Shifts_duration_max_hours = 2
+Payform_item_max_hour = 3
+
+Total_days = Ndays_of_schedule_into_future+Ndays_of_schedule_into_past
+
+# Custom Ratios
+Active_notice_chance = 0.8
+Public_loc_group_chance = 0.6
+User_with_nick_name_chance = 0.3
+User_profile_complete_chance = 0.9
+User_has_pic_chance = 0.4
+Location_has_time_slot_chance = 0.8
+Weekday_has_time_slot_chance = 5/7.0
+Shift_signed_in_chance = 0.9
+Payform_submitted_chance = 0.9
+Payform_printed_chance = 0.7
```

Comment:
Why are all these variables capitalized? That's usually reserved for class names.